### PR TITLE
chore(coverage): add codecov.io integration (WIP)

### PR DESCRIPTION
**Pull request by arschles**

### CODE REVIEW

File: test-cover.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+echo "" > coverage.txt
+
+for d in $(find . -maxdepth 10 -type d); do
```
Comment (mboersma) [inline test-cover.sh]: What about:
```bash
for d in $(go list $(glide novendor)); do
```

**Comment by mboersma** [comment]
See also #314.

**Comment by arschles** [comment]
bumping from RC1, as this is not critical for the RC

**Comment by arschles** [comment]
Closing in favor of #314

**Pull request closed without merging**